rpcs3/Emu/Cell/lv2/sys_net/lv2_socket_p2p.cpp

```diff
@@ -390,13 +395,17 @@ std::tuple<bool, bool, bool> lv2_socket_p2p::select(bs_t<lv2_socket::poll_t> sel
 	// Check if it's a bound P2P socket
 	if ((selected & lv2_socket::poll_t::read) && vport && !data.empty())
 	{
-		sys_net.trace("[P2P] p2p_data for vport %d contains %d elements", vport, data.size());
+		if (!data.empty())
```

Comment:
This scope can only be entered if data is not empty anyway, so this is a redundant check